fdbrpc/crc32c.cpp

```diff
@@ -38,53 +38,6 @@
 #include "generated-constants.cpp"
 #pragma GCC target("sse4.2")
 
-static uint32_t append_trivial(uint32_t crc, const uint8_t * input, size_t length)
```

Comment:
I believe this file was not written by us, so we should probably leave the functions